Formula/copilot.rb

```diff
@@ -24,6 +24,7 @@ def install
 
     system "make", "tools"
     system "make", "package-custom-resources"
+    system "make", "packr-build"
```

Comment:
Does this package it's own versions of tools? Or does it use Homebrew's?

Reply:
The package uses its own versions of tools as defined here: https://github.com/aws/copilot-cli/blob/d709cfefb5c40d4e63e4e47141f25658e5e7b9e1/Makefile#L114-L119

Reply:
There is a packer formula, it should use the homebrew version of the formula instead of downloading unversioned things.